Backport patch from 6.6.0 to only require backports.functools_lru_cache for
Python < 3.3

https://github.com/cherrypy/cheroot/commit/18447cbee7135f3f5017c8f6a9607dc1f7259570

--- setup.cfg.orig	2019-08-18 22:27:10 UTC
+++ setup.cfg
@@ -57,7 +57,7 @@ setup_requires = 
 	setuptools_scm>=1.15.0
 	setuptools_scm_git_archive>=1.0
 install_requires = 
-	backports.functools_lru_cache
+	backports.functools_lru_cache ; python_version < '3.3'
 	six>=1.11.0
 	more_itertools>=2.6
 
